### PR TITLE
Feat: support download pod lods in velaux

### PR DESCRIPTION
**Pull request by basuotian**
### Description of your changes

Close: #720 

preview:
image

image

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
- [ ] Run `yarn lint` to ensure the frontend changes are ready for review.
- [ ] Run `make reviewable`to ensure the server changes are ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: packages/velaux-ui/src/pages/ApplicationInstanceList/components/ContainerLog/index.tsx [hunk 1]
```diff
@@ -126,7 +142,18 @@ class ContainerLog extends Component<Props, State> {
...
         v2
-        title={<Translation>Container Log</Translation>}
+        title={
+          <Row style={{ width: '100%' }}>
+            <Col span={12}>
+              <Translation>Container Log</Translation>
+            </Col>
+            <Col span={12}>
+            <Button style={{float: "right"}} type="normal" size="small" onClick={this.downloadLog}>
+              <Translation className="font-bold font-size-14">download</Translation>
```
Comment (barnettZQG) [inline packages/velaux-ui/src/pages/ApplicationInstanceList/components/ContainerLog/index.tsx:152]: Let's use the AiOutlineDownload icon to replace the text.

File: packages/velaux-ui/src/pages/ApplicationLog/components/LogContainer/index.tsx [hunk 2]
```diff
@@ -138,12 +138,31 @@ class ContainerLog extends Component<Props, State> {
     }
   };
 
+  downloadLog = () => {
```
Comment (barnettZQG) [inline packages/velaux-ui/src/pages/ApplicationLog/components/LogContainer/index.tsx:141]: This code is repetitive, let's make it reusable.

File: packages/velaux-ui/src/pages/ApplicationInstanceList/components/ContainerLog/index.tsx (hunk 1 above)
Comment (basuotian) [reply packages/velaux-ui/src/pages/ApplicationInstanceList/components/ContainerLog/index.tsx]: Done, and the preview is updated

File: packages/velaux-ui/src/locals/Zh/zh.json [hunk 3]
```diff
@@ -622,5 +622,6 @@
   "Show Values File": "查看 Value 文件",
   "Showing all projects you are permitted": "你拥有权限的所有项目",
   "More": "显示更多",
-  "Hide": "收起"
+  "Hide": "收起",
```
Comment (barnettZQG) [inline packages/velaux-ui/src/locals/Zh/zh.json:625]: This is unuseful, right?

Comment (basuotian) [reply packages/velaux-ui/src/locals/Zh/zh.json]: yes, i'll remove it

**Review by barnettZQG — APPROVED** [review]
LGTM

**Pull request merged**